build.gradle

```diff
@@ -436,14 +436,21 @@ String getVersionString() {
     }
 
     def version_hash = ''
+    def branch_name = ''
     if (!build_server) {
         try {
-            version_hash = "git rev-parse --short HEAD".execute().text.trim()
+            version_hash = "-" + "git rev-parse --short HEAD".execute().text.trim()
+        } catch (ignored) {
+        }
+
+        try {
+            branch_name = "git rev-parse --abbrev-ref HEAD".execute().text.trim()
+            branch_name = "-" + branch_name.substring(branch_name.lastIndexOf('/') + 1)

```

Comment:
This means dev builds include the name of the branch (after the last `/`), it was so that we could differentiate between different dev builds off different places :)